src/main/resources/db/migration/V3_5__add_original_aktivitetskrav_uuid.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE AKTIVITETSKRAV
+    ADD COLUMN original_aktivitetskrav_uuid UUID;
```

Comment:
`UUID`-type var ny for meg. Tror vi har pleid å bruke `CHAR(36)`, men dette funker sikkert også :smiley:

Reply:
`UUID` var gyldig i postgresql iallefall 😊 Finnes det en egen type for det stemmer jeg for at vi bruker det 😁